src/clj_kondo/impl/linters.clj

```diff
@@ -501,13 +515,14 @@
                   :end-row (:end-row binding)
                   :end-col (:end-col binding)}))))
           (doseq [default defaults
-                  :let [binding (:binding default)]
+                  :let [binding (:binding default)
+                        nm (:name binding)]
```

Comment:
This seems to be an unnecessary change (keyword lookup can be prevented when linter doesn't apply).